app.js

```diff
@@ -64,12 +64,12 @@ function save() {
 
 function load() {
   let parsedProducts = localStorage.getItem("productData");
-  JSON.parse(parsedProducts);
+  parsedProducts = JSON.parse(parsedProducts);
+  parsedProducts = allProducts;
 
-  for (let i = 0; i < allProducts.length; i++) {
-    let product = allProducts[i];
+  for (let i = 0; i < parsedProducts.length; i++) {
+    let product = parsedProducts[i];
     loadedProducts.push(product);
-
   }
   savedProducts = JSON.stringify(loadedProducts);
   localStorage.setItem("productData", savedProducts);
```

Comment:
On a separate note, `localStorage.setItem()` appears only twice in this document, once in `save()` and once in `load()` Because of the way those two functions are coded, you're only going to access local storage at the BEGINNING of the user experience. With your site coded that way, you never save any user data.